conduct.html

```diff
@@ -87,6 +87,12 @@ <h2>Scope</h2>
         account, or acting as an appointed representative at an online or
         offline event. Representation of a project may be further defined and
         clarified by project maintainers.
+
+        The project spaces under this Code of Conduct are:
```

Comment:
Feels like we're redundantly restating what's clearly stated above, and limiting project spaces more that necessary.

Should we be pointing out that there are non-sanctioned spaces related to Rails that are not "project spaces" where this code of conduct does not apply?